pkg/executor/terraform/terraform.go

```diff
@@ -57,6 +58,42 @@ func Init() (string, error) {
 	return output, nil
 }
 
+func SwitchWorkspace(name string) error {
+	var workspaceList = shell.Command{
+		WorkingDir: getWorkingDir(),
+		Command:    "terraform",
+		Args:       []string{"workspace", "list"},
+	}
+	var workspaceCreate = shell.Command{
+		WorkingDir: getWorkingDir(),
+		Command:    "terraform",
+		Args:       []string{"workspace", "new", name},
+	}
+	var workspaceSelect = shell.Command{
+		WorkingDir: getWorkingDir(),
+		Command:    "terraform",
+		Args:       []string{"workspace", "select", name},
+	}
+	output, err := shell.Execute(workspaceList)
+	if err != nil {
+		return err
+	}
+	// no workspace is created yet
+	if !strings.Contains(string(output), name) {
+		_, err = shell.Execute(workspaceCreate)
+		if err != nil {
+			return err
+		}
+	//selec workspace
+	} else {
+		_, err = shell.Execute(workspaceSelect)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
```

Comment:
Please try to avoid [else blocks](https://youtu.be/5DVV36uqQ4E?t=662). Can be refactored to something like this:

```suggestion
    if !strings.Contains(string(output), name) {
		_, err = shell.Execute(workspaceCreate)
		return err
    }
     _, err = shell.Execute(workspaceSelect)
    return err
```

Reply:
applied